CoreFoundation/Base.subproj/CFUtilities.c

```diff
@@ -1000,6 +1000,7 @@ void CFLog1(CFLogLevel lev, CFStringRef message) {
         CFStringGetCString(message, buffer, maxLength, encoding);
     
     __android_log_print(priority, tag, "%s", buffer);
+    fprintf(stderr, "%s\n", buffer);
```

Comment:
What is this for? Isn't `__android_log_print` enough?

Reply:
This is helpful when you are running an Android binary (such as the tests) from the command line using an “adb shell". Otherwise NSLogs are only visible using an “adb logcat” by which time the messages have lost their context. This doesn’t affect normal running of an actual application and the logging to stderr goes into the void.